.gitignore

```diff
@@ -1,27 +1,27 @@
-*.pydevproject
-.project
-.metadata
-bin/**
-tmp/**
-tmp/**/*
-*.tmp
-*.bak
-*.swp
-*~.nib
-local.properties
-.classpath
-.settings/
-.loadpath
-target/
-test-output/
-.externalToolBuilders/
-*.launch
-.cproject
-.buildpath
-.springBeans
-cas.log
-perfStats.log
-*.iml
-.idea
-overlays
-pac4j-test-cas/overlays/
+*.pydevproject

```

Comment:
As I said.. When I did the merge I  already had checked in what I had with whatever formatting changes my formatter used.. At the point of doing the merge I just merged. To fix these individual files I guess the best plan is to look at the pull request and for any file where there are no actual changes just replace my version with it?